### PR TITLE
Tweak node capacity overprovisioning task

### PR DESCRIPTION
**Pull request by sftim**
Tweaks to recently-merged task page https://kubernetes.io/docs/tasks/administer-cluster/node-overprovisioning/

[preview](https://deploy-preview-48753--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/node-overprovisioning/)

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
/lgtm

**Comment by Shubham82** [comment]
Thanks @sftim,
The preview looks good to me.
https://deploy-preview-48753--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/node-overprovisioning/

/lgtm

**Comment by sftim** [comment]
I revised now that https://github.com/kubernetes/website/issues/41943 has merged.

File: content/en/examples/priorityclass/low-priority-class.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: placeholder
+  name: placeholder # these Pods represent placeholder capacity
 value: -1000
 globalDefault: false
 description: "Negative priority for placeholder pods to enable overprovisioning."
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/examples/priorityclass/low-priority-class.yaml:7]: Which red error icon, sorry?

[the remark this answers is not in the document]
Comment (sftim) [reply content/en/examples/priorityclass/low-priority-class.yaml:7]: Ah, it's not really an error and also the existing file doesn't have a final newline.

[the remark this answers is not in the document]
Comment (network-charles) [reply content/en/examples/priorityclass/low-priority-class.yaml:7]: Oh, ok.

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**